bootiso

```diff
@@ -772,11 +784,15 @@ mountISOFile() {
 # $1 : a device block
 # Return 0 if device is USB, 1 otherwise
 getDeviceType() {
-	typeset deviceName=/sys/block/${1#/dev/}
-	typeset deviceType=$(udevadm info --query=property --path="$deviceName" | grep -Po 'ID_BUS=\K\w+')
+	typeset deviceType=$(udevadm info --query=property "/dev/$device" | grep -Po 'ID_BUS=\K\w+')
 	echo "$deviceType"
 }
 
+deviceIsThumb() {
```

Comment:
I would also document that it echoes empty or `'1'` string ; as one might check against `true` or `false` inadvertently. 
Or you can normalize the value to `true` / `false` before echo-ing

Reply:
I've normalized to '1' and '0'.